src/lib/util/tauri.ts

```diff
@@ -50,3 +50,13 @@ export async function resetAndClear({
   // this may throw error that file does not exist if config file is in the same directory as plot files (removed on the line above)
   await config.remove();
 }
+
+export async function writeFile(
```

Comment:
it seems we only use it in the `Config` class, so maybe we could use `invoke` inside `Config.write` instead of declaring this funciton in `utils` and injecting it into `Config`?

Reply:
Nevermind, we can keep it separate and inject (to keep it testable)